ckan_cloud_operator/providers/db/proxy/azuresql/manager.py

```diff
@@ -0,0 +1,156 @@
+#
+
+import os
+import subprocess
+
+from distutils.util import strtobool
+
+from ckan_cloud_operator import kubectl
+from ckan_cloud_operator import logs
+
+from ckan_cloud_operator.providers.cluster import manager as cluster_manager
+from ckan_cloud_operator.config import manager as config_manager
+
+
+def initialize(db_prefix=None):
+    _apply_service(db_prefix)
+    _apply_deployment(db_prefix)
+    if not db_prefix:
+        _set_provider()
+    else:
+        _config_interactive_set({
+            'port-forwarded-port': '5433'
+        }, suffix=db_prefix)
+        db_prefixes = _config_get('all-db-prefixes')
+        db_prefixes = db_prefixes.split(',') if db_prefixes else []
+        if db_prefix not in db_prefixes:
+            db_prefixes.append(db_prefix)
+            _config_set('all-db-prefixes', ','.join(db_prefixes))
+
+
+def get_all_db_prefixes():
+    prefixes = _config_get('all-db-prefixes')
+    if prefixes:
+        return prefixes.split(',')
+    else:
+        return []
+
+
+def update(**kwargs):
+    pass
+
+
+def get_internal_proxy_host_port(db_prefix=None):
+    namespace = cluster_manager.get_operator_namespace_name()
+    service_name = _get_resource_name(suffix=db_prefix or '')
+    return f'{service_name}.{namespace}', 5432
+
+
+def get_external_proxy_host_port(db_prefix=None):
+    if strtobool(os.environ.get('CKAN_CLOUD_OPERATOR_USE_PROXY', 'y')):
+        host, port = get_external_proxy_forwarded_host_port(db_prefix)
+    else:
+        host, port = None, None
+    return host, port
+
+
+def get_external_proxy_forwarded_host_port(db_prefix=None):
+    return '127.0.0.1', get_port_forwarded_port(db_prefix)
+
+
+def get_port_forwarded_port(db_prefix=None):
+    if db_prefix:
+        return _config_get(key='port-forwarded-port', suffix=db_prefix or '')
+    else:
+        return 5432
+
+
+def start_port_forward(db_prefix=None):
+    """Starts a local proxy to the cloud SQL instance"""
+    print("\nKeep this running in the background\n")
+    namespace = cluster_manager.get_operator_namespace_name()
+    deployment_name = _get_resource_name(suffix=db_prefix or '')
+    port = get_port_forwarded_port(db_prefix)
+    subprocess.check_call(f'kubectl -n {namespace} port-forward deployment/{deployment_name} {port}:5432',
+                          shell=True)
+
+
+def _apply_deployment(db_prefix=None):
+    db_host = config_manager.get(secret_name='ckan-cloud-provider-db-azuresql-credentials')['azuresql-host']
+    app_name = _get_resource_labels(for_deployment=True, suffix=db_prefix or '').get('app')
+    kubectl.apply(kubectl.get_deployment(
+        _get_resource_name(suffix=db_prefix),
+        _get_resource_labels(for_deployment=True, suffix=db_prefix or ''),
+        {
+            'replicas': 1,
+            'revisionHistoryLimit': 10,
+            'strategy': {'type': 'RollingUpdate', },
+            'selector': {
+                'matchLabels': {
+                    'app': app_name
```

Comment:
please use the proper method for getting the labels

Reply:
done